src/almanack/metrics/data.py

```diff
@@ -507,9 +525,7 @@ def compute_repo_data(repo_path: str) -> None:
             else None
         ),
         "repo-days-between-last-coverage-run-latest-commit": (
-            (most_recent_commit_date - date_of_last_coverage_run).strftime(
-                "%Y-%m-%dT%H:%M:%S.%fZ"
-            )
+            (most_recent_commit_date - date_of_last_coverage_run).days
```

Comment:
Nice catch; sorry that I missed that in the last review! I'll pay more attention to the labels for your dict elements and whether the value matches going forward.